### ISSUE TITLE
iOS Version

### ISSUE THREAD

**Issue by KintCark** [ShiftHackZ/Stable-Diffusion-Android#356]
Is it possible to get this app on iOS?

**Comment by ShiftHackZ** [comment]
App development for iOS is more complicated, so to get this app on iOS, IMHO we need to complete at least this action items:

- Make a deep investigation if all of app features can be implemented on iOS (I think LocalDiffusion generation providers may not be possible, but I may be wrong here).
- Find an iOS developer who is interested in implementing and maintaining the same code-base for iOS (I am not iOS developer myself).
- Need to find funding for Apple hardware (Mac computer and iPhone) for the iOS developer, because iOS development is only possible on Mac and debugging this kind of app most likely will require real phone. 
- Need to find funding at least $100/per year to pay for Apple Development program fee.

To sum up: for the small non-profit app which is highly specialized and has no monetization model, I don't believe it is possible.
